src/main/java/com/bookstore/exception/message/ExceptionMessage.java

```diff
@@ -0,0 +1,13 @@
+package com.bookstore.exception.message;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum ExceptionMessage {
```

Comment:
delete it. In a real app you can have 1000 entities. So, you will have 1000 enum values only for "not found"